build.gradle

```diff
@@ -931,6 +936,20 @@ task runQuickstart {
       if (conffile.exists())
         conffile.delete()
     }
+    //Setting _JAVA_OPTIONS to avoid using gradle settings which is 256m which was causing rdd to
+    // overflow on disk
+    Map<String,Object> env = new HashMap<String,Object>()
+    env.put("_JAVA_OPTIONS","-Xms256m -Xmx4g")
```

Comment:
Ok, but why cannot both of them be equal? When they are not equal then GemXD ResourceManager will not start up which can cause trouble.